src/web/core/src/main/java/org/geoserver/web/wicket/CodeMirrorEditor.js

```diff
@@ -3,7 +3,8 @@ var editor = CodeMirror.fromTextArea(textarea, {
     mode: '$mode',
     theme: 'default',
     lineWrapping: true,
-    lineNumbers: true
+    lineNumbers: true,
+    extraKeys: {"Ctrl-Space": "autocomplete"}
```

Comment:
Looks like Ctrl-Space is still used (an works fine) on OS X in those examples, so I guess that is fine.